Lib/mutatorMath/objects/mutator.py

```diff
@@ -29,6 +29,7 @@ def buildMutator(items, axes=None, bias=None):
         bender = noBend
     # the order itself does not matter, but we should always build in the same order.
     items = sorted(items)
+    #items = sorted(items)
```

Comment:
why did you comment this out?

Reply:
Hm that was an older change that should not be there.